x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -546,6 +545,7 @@ export async function createInstallation(options: {
     ? true
     : undefined;
 
+  // TODO cleanup removable flag and isUnremovablePackage function
```

Comment:
@mostlyjason what about the endpoint package, can it be removable as well?

Reply:
I don't see why not. Adding @kevinlog for confirmation. For context, we are trying to remove the concept of unremovable packages since they will no longer be installed by default. This would also allow users to reinstall the package if there is a problem. Can you think of any reason users shouldn't be able to remove endpoint?

Reply:
@mostlyjason @juliaElastic I think it's OK to make the endpoint package removable especially for reasons you stated such as allowing users to uninstall and reinstall if there is a problem.

If the user removed all integration policies first then all of their Endpoints would be uninstalled.  I would just like to test what would happen with their data if they uninstalled the package.  This would imply that we remove mappings, etc.  If we're going to make the Endpoint package removable, I'd like to open up a separate ticket/PR if possible where we can test a few different scenarios before merging/releasing.

Reply:
@kevinlog thanks for your response, are you comfortable with testing this soon as we are planning to merge this feature for 8.1? If not, should we keep endpoint package unremovable for now?
cc @joshdover

Reply:
When we uninstall packages, I don't believe any existing data should be affected, though the next rollover of the data stream would cause the write index to pickup the generic mappings for `logs-*-*` and `metrics-*-*` rather than the endpoint-specific ones. This would only affect new data that is ingested which shouldn't be happening once a package is uninstalled anyways.